test/specs/collections/Form/FormField-test.js

```diff
@@ -11,7 +11,9 @@ describe('FormField', () => {
   common.propKeyOnlyToClassName(FormField, 'error')
   common.propKeyOnlyToClassName(FormField, 'disabled')
   common.propKeyOnlyToClassName(FormField, 'inline')
-  common.propKeyOnlyToClassName(FormField, 'required', { label: '' })
+  common.propKeyOnlyToClassName(FormField, 'required', {
+    requiredProps: { label: '' },
+  })
```

Comment:
Updated all tests that were using the `requiredProps` last arg to now use the `options` object param.